src/test/java/study/string/SplitTest.java

```diff
@@ -0,0 +1,55 @@
+package study.string;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import org.junit.jupiter.api.Test;
+
+public class SplitTest {
+
+
+    @Test
+    public void 숫자_두개_분리_테스트(){
+        String[] split = "1,2".split(",");
+        assertThat(split).hasSize(2);
+        assertThat(split).contains("1","2");
+        assertThat(split).containsExactly("1","2");
```

Comment:
```suggestion
        String[] split = "1,2".split(",");

        assertThat(split).containsExactly("1", "2");
```
`containsExactly`만으로도 나머지 2개 케이스까지 검증할 수 있을 것 같아요